html/index.html.haml

```diff
@@ -81,6 +81,20 @@
           = render_scope scope
 
         .hidden#inspection_details
+          - if @description[:environment].attributes.include?("system_type")
```

Comment:
You can also access the system_type with just `@description[:environment].system_type`.